static-main/js/form.js

```diff
@@ -13,13 +13,13 @@ if ($.validate) {
     validateOnEvent: true,
     onError: function($form) {
       $('form[id=' + $form.attr('id') +'] .generic-error').removeClass('d-none');
-  },
+    },
```

Comment:
whitespace included here.

Reply:
I don't understand.

Reply:
No I added the whitespace myself. Previously the indent level did not match on this line, every other line started at the 3rd level and this line was not indented.